tests/sentry/replays/test_organization_replay_index.py

```diff
@@ -550,7 +553,6 @@ def test_get_replays_user_filters(self):
                 # Tag filters.
                 "tags[a]:m",
                 "a:m",
-                "a:[n,o]",
```

Comment:
this is the only behavior change in this PR -- you can no longer search across the segments for values of tags,  only the first. so if your tag is not on scope when the replay starts, it will not be available for filtering

Reply:
I think this is fine.  We should stop ingesting tags on `segment_id > 0`.  I will follow up on that.